docs/src/docs/common-pitfalls.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Common pitfalls
+---
+
+For the most part, you should be able to seamlessly migrate your site from 11ty to Slinkity with minimal reconfiguration. But there are a few changes that may catch you off guard. These gotchas are summarized below.
+
+## Certain files disappeared from my production build output folder
+
+In an 11ty site, you are free for the most part to structure your project however you like. You can put your static assets in a `public` folder outside your [11ty input directory]((https://www.11ty.dev/docs/config/#input-directory)) and pass-through copy them, but you don't need to—you could also store all of your assets inside your 11ty input directory. It doesn't matter which approach you take, so long as you configure your 11ty project accordingly and write your files to the correct locations. However, you need to take a slightly different approach in Slinkity due to the standards enforced by Vite.
+
+First, we need to understand that a Slinkity build consists of multiple steps:
+
+1. Processing your site with 11ty and dumping the result into a temporary build folder (e.g., `.11ty-build-DcNVBN`).
+2. Processing the temporary build folder with Vite to bundle assets on the pages that reference them.
+3. Writing the temporary build folder to your 11ty output directory (e.g., `_site`)
+4. Deleting the temporary build folder.
+
+Vite is very strict about where it wants you to place your static assets—in a [dedicated `public` folder](https://vitejs.dev/guide/assets.html#the-public-directory). This allows it to distinguish static assets from ones that are referenced by your pages (like stylesheets) and to bundle the referenced assets while leaving the static ones alone. Any static asset that isn't referenced by a page and isn't in the `public` directory will be removed from the final build output. For this reason, you may find that certain files disappear from production builds, even though they were technically processed correctly by 11ty.
+
+Below are some scenarios where you may encounter this pitfall:
+
+- Self-hosted font files
```

Comment:
This is a _bit_ misleading I'm afraid. For instance, fonts actually don't need to go in the `public` folder, so long as they're referenced from some other file using a relative path or [import alias](https://deploy-preview-114--slinkity.netlify.app/docs/import-aliases/). For instance, If I have a CSS file that applies the `@font-face` like so:

```css
@font-face {
  font-family: Atkinson;
  src: url('./fonts/Atkinson/Atkinson-Hyperlegible-Regular.woff2'),
      url('./fonts/Atkinson/Atkinson-Hyperlegible-Regular.woff'),
      url('./fonts/Atkinson/Atkinson-Hyperlegible-Regular.ttf');
  font-display: swap;
}
```

...they'll get copied to the build output no problem! The problem only arrises for resources that _no one references via a relative path or import alias._

That admittedly wasn't clear to me after reading this section. I'd honestly suggest adding the example I listed above to explain what _does_ work.

Reply:
Also worth stating:
1. You can configure `/public` to be any directory you like using a Vite Config (see [config options](https://deploy-preview-114--slinkity.netlify.app/docs/config/#vite's-vite.config.js))
2. You'll need to [passthrough copy](https://www.11ty.dev/docs/copy/) your public directory for it to exist in the final build output

Reply:
What are your thoughts on us splitting the explanations for the `public` folder out into a separate page? Too heavy handed?

Reply:
Decision: split out to new "how we build your site" page. Then, add further sections on managing font faces as an example